cypress/integration/codeName_Dolphin/Keyboard_spec.js

```diff
@@ -39,6 +38,7 @@ describe('Keyboard', () => {
   })
 
   it('Should toggle key values on and off when clicked', () => {
+    cy.viewport(1920, 975) 
```

Comment:
thanks for making this a little smarter, this wasn't working for me originally, but now I see why!